config/etc/system-image/writable-paths

```diff
@@ -42,35 +33,29 @@
 # needed by apparmor - use transition since some core apps are
 # pre-installed on the image
 /var/cache/apparmor                     auto                    persistent  transition  none
+# FIXME: do we still need this?
 /var/lib/apparmor                       auto                    persistent  transition  none
```

Comment:
i see an empty profile/ dir created in there on my installs, as long as nothing fails when it cant create that dir we should be fine